docs/KlassenOffentligTjeneste.adoc

```diff
@@ -697,9 +697,9 @@ _Norwegian extension: Range changed from `cpsv:PublicService` to `cpsvno:Service
 | _English name_ | _sector_
 | URI | cv:sector
 | Verdiområde / _Range_ | skos:Concept
-| Anvendelse / _Usage note_ |  Egenskapen brukes til å referere til industri/sektor som den aktuelle offentlige tjenesten er relatert til, eller er ment for. En tjeneste kan relateres til flere industrier/sektorer.
+| Anvendelse / _Usage note_ |  Egenskapen brukes til å referere til industri/sektor som den aktuelle offentlige tjenesten er relatert til, eller er ment for. En tjeneste KAN relateres til flere industrier/sektorer.
 
-_This property represents the industry or sector a Public Service relates to, or is intended for. Note that a single Public Service may relate to multiple sectors._
+_This property represents the industry or sector a Public Service relates to, or is intended for. Note that a single Public Service MAY relate to multiple sectors._
 | Multiplisitet / _Multiplicity_ | 0..n
 | Kravnivå / _Requirement level_ | Valgfri / _Optional_
 | Merknad / _Note_ | De mulige verdiene for egenskapen er beskrevet i EUs kontrollerte vokabular https://op.europa.eu/en/web/eu-vocabularies/concept-scheme/-/resource?uri=http://publications.europa.eu/resource/authority/data-theme[Data theme &#x29C9;, window="_blank", role="ext-link"].
```

Comment:
Verdien SKAL velges fra EUs kontrollerte vokabular.... ? og
The value MUST be chosen from EU's controlled vocabulary...?

Reply:
Kopiert og oversatt fra EU. Rettes for å være konsekvent.